### ISSUE TITLE
Chat tabability not visualized or missing (accessibility)

### ISSUE THREAD

**Issue by bcullman** [mckaywrigley/chatbot-ui#245]
much of the interactive chat UI is either missing visual cues for focus, or left out of the tab order entirely. Here are some examples:

https://user-images.githubusercontent.com/5056972/228094371-c0c5f2b3-38a2-43f9-8fda-8a57db277722.mp4
In the above video, we can see a user using the tab key to navigate through the intial UI for the chat interface. while the tab key is advancing to the next selected element, it is not visualized until we get to the last link on the page ("Chatbot UI")

https://user-images.githubusercontent.com/5056972/228095519-571de3f0-12d1-4c73-bffe-ab8f3cdf2cba.mp4
In the above video, a chat is in progress. while tabbing from the top of the page, the tab key completely misses the gear icon (settings), the slanted 3 lines icon ("clear all messages") the edit icon (for each messages) the copy icon (for AI responses). Tab does stop on the codeblock "copy" and "download" buttons, but the tab stops are not visualized. the prompt selector also overflow the it's container.
